solutions/usaco-1061.mdx

```diff
@@ -106,4 +106,137 @@ for x in eat:
 
 </PySection>
 
+<JavaSection>
+```java
+
+import java.io.*;
+import java.util.*;
+
+public class StuckInARut {
+	static BufferedReader r = new BufferedReader(
+		new InputStreamReader(System.in));
```

Comment:
I split this because @sathu-beep noted that it's above 80 characters. Should I revert this commit then?

I wanted to follow the code sample listed in [this file](https://github.com/cpinitiative/usaco-guide/blob/master/content/1_General/Input_Output.mdx/), hence the static readers and writers. I'll move it into main tho! :D

Reply:
that smiley face slightly unsettles me

Reply:
:D